editor/gui/editor_spin_slider.cpp

```diff
@@ -604,6 +609,10 @@ void EditorSpinSlider::_grabber_mouse_exited() {
 
 void EditorSpinSlider::set_read_only(bool p_enable) {
 	read_only = p_enable;
+	if (read_only && value_input && value_input->has_focus()) {
```

Comment:
```suggestion
	if (read_only && value_input) {
```
It's already checked in `release_focus()`.